azure/functions/decorators/function_app.py

```diff
@@ -713,38 +715,41 @@ def decorator():
-                          lease_renew_interval: Optional[int] = None,
-                          lease_acquire_interval: Optional[int] = None,
-                          lease_expiration_interval: Optional[int] = None,
-                          max_items_per_invocation: Optional[int] = None,
-                          start_from_beginning: Optional[bool] = None,
-                          preferred_locations: Optional[str] = None,
-                          data_type: Optional[
-                              Union[DataType, str]] = None,
-                          **kwargs: Any) -> \
+    def cosmos_db_trigger_v3(self,
+                             arg_name: str,
+                             database_name: str,
+                             collection_name: str,
+                             connection_string_setting: str,
+                             lease_collection_name: Optional[str] = None,
+                             lease_connection_string_setting: Optional[
+                                 str] = None,
+                             lease_database_name: Optional[str] = None,
+                             create_lease_collection_if_not_exists: Optional[
+                                 bool] = None,
+                             leases_collection_throughput: Optional[int] =
+                             None,
+                             lease_collection_prefix: Optional[str] = None,
+                             checkpoint_interval: Optional[int] = None,
+                             checkpoint_document_count: Optional[int] = None,
+                             feed_poll_delay: Optional[int] = None,
+                             lease_renew_interval: Optional[int] = None,
+                             lease_acquire_interval: Optional[int] = None,
+                             lease_expiration_interval: Optional[int] = None,
+                             max_items_per_invocation: Optional[int] = None,
+                             start_from_beginning: Optional[bool] = None,
+                             preferred_locations: Optional[str] = None,
+                             data_type: Optional[
+                                 Union[DataType, str]] = None,
+                             **kwargs: Any) -> \
             Callable[..., Any]:
-        """The cosmos_db_trigger decorator adds :class:`CosmosDBTrigger`
+        """The cosmos_db_trigger_v3 decorator adds :class:`CosmosDBTrigger`
         to the :class:`FunctionBuilder` object
         for building :class:`Function` object used in worker function
-        indexing model. This is equivalent to defining CosmosDBTrigger
-        in the function.json which enables function to be triggered when
-        CosmosDB data is changed.
+        indexing model. This decorator will work only with extension bundle 2.x
+        or 3.x. For additional details, please refer
+        https://github.com/Azure/azure-functions-python-worker/issues/1222.
```

Comment:
Why not put an aka.ms link or mslearn link?  aka.ms can be tracked.

Reply:
Added